cmd/bridge/main.go

```diff
@@ -123,6 +123,7 @@ func main() {
 
 	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "Allow catalog categories customization. (JSON as string)")
 	fUserSettingsLocation := fs.String("user-settings-location", "configmap", "DEV ONLY. Define where the user settings should be stored. (configmap | localstorage).")
+	fProjectAccess := fs.String("project-access", "", "The list of Cluster Roles assignable for the project access page. (JSON as string)")
```

Comment:
I'd give this a more specific name, maybe `project-access-cluster-roles`.

I'm tempted to ask that you put `-json` in the name, but we didn't do it for `developer-catalog-categories`. So I guess it's better not to so we're consistent.

Reply:
Updated to `project-access-cluster-roles`.